lib/blockTemplate.js

```diff
@@ -226,21 +235,39 @@ var BlockTemplate = module.exports = function BlockTemplate(
             });
         }
 
-        /*
-        console.log('header: ' + header.toString('hex'));
-        console.log('soln: ' + soln.toString('hex'));
-        console.log('varInt: ' + varInt.toString('hex'));
-        console.log('this.genTx: ' + this.genTx);
-        console.log('data: ' + value.data);
-        console.log('buf_block: ' + buf.toString('hex'));
-        */
+
+        if (this.rpcData.version == 3 && coin.symbol.indexOf("zen") !== -1) {
```

Comment:
add more specific if instead of indexOf("zen") - here and in the other similar if